srv/salt/ceph/monitoring/prometheus/README.md

```diff
@@ -0,0 +1,74 @@
+# Monitoring
+By default DeepSea deploys a monitoring stack on the salt master. This consists
+of prometheus and Grafana, as well as the ceph_exporter of the salt master and
```

Comment:
prometheus (initial lower) and Grafana (initial upper) -> on purpose?

Reply:
For some reason I thought that was the preferred way...not sure why and it isn't.